Misc/NEWS.d/next/Core and Builtins/2018-02-07-10-55-58.bpo-1100942.md4agI.rst

```diff
@@ -0,0 +1 @@
+Add datetime.date.strptime and datetime.time.strptime class methods.
```

Comment:
To get links in the HTML rendered Changelog:

```suggestion
Add :func:`datetime.date.strptime` and :func:`datetime.time.strptime` class methods.
```

Reply:
Can you please add an entry in Doc/whatsnew/3.8.rst as well? IHMO it deserves to be documented there.